cmake/std/atdm/ATDMDevEnvSettings.cmake

```diff
@@ -222,16 +222,12 @@ IF (ATDM_FPIC)
   SET(CMAKE_POSITION_INDEPENDENT_CODE ON CACHE BOOL "Build targets with position independent code")
 ENDIF()
 
-ATDM_SET_CACHE(CMAKE_C_FLAGS
-  "$ENV{ATDM_CONFIG_C_FLAGS} $ENV{EXTRA_C_FLAGS}" CACHE STRING)
-ATDM_SET_CACHE(CMAKE_CXX_FLAGS
-  "$ENV{ATDM_CONFIG_CXX_FLAGS}${EXTRA_EXTRA_CXX_FLAGS} $ENV{EXTRA_CXX_FLAGS}" CACHE STRING)
-ATDM_SET_CACHE(CMAKE_Fortran_FLAGS
-  "$ENV{ATDM_CONFIG_Fortran_FLAGS} $ENV{EXTRA_Fortran_FLAGS}" CACHE STRING)
-# NOTE: That above "EXTRA_<LANG>_FLAGS" env vars are left in for backward
-# compatibility with the EMPIRE configuration and are not directly set in any
-# offical ATDM Trilinos configuration setup.  Only the
-# "ATDM_CONFIG_<LANG>_FLAGS" env vars are set.
+SET(CMAKE_C_FLAGS
+  "$ENV{ATDM_CONFIG_C_FLAGS} ${CMAKE_C_FLAGS}")
+SET(CMAKE_CXX_FLAGS
+  "$ENV{ATDM_CONFIG_CXX_FLAGS}${EXTRA_EXTRA_CXX_FLAGS} ${CMAKE_CXX_FLAGS}")
```

Comment:
do you need a space in between the first and second arguments on this line?

Reply:
> do you need a space in between the first and second arguments on this line?

@rppawlo, not in this case given how that var is created above this.  But to be super safe, I guess it might be a good idea to put in a space.

I will push a commit to add that space to be safe.

Reply:
> I will push a commit to add that space to be safe.

@rppawlo, I just pushed https://github.com/trilinos/Trilinos/pull/9092/commits/3dc3b5019280cd90604ed6d5c199b14e31baa3f7